Automattic/wp-calypso
# Gravatar: Align design and consist code for the signup page

Related to 105630-gh-Automattic/gravatar

## Proposed Changes

* To align design
* To use Gravatar logo from `oauth2Client.icon`
* To consist code with the up coming new Gravatar passwordless login flow (105630-gh-Automattic/gravatar)

## Testing Instructions



* Setup Calypso by following [this doc](https://github.com/Automattic/wp-calypso#getting-started)
* Check out this PR
* Run `yarn install` and then `yarn start`
*  Logout your WP.com account and then go to the [updated signup page](http://calypso.localhost:3000/start/wpcc/oauth2-user?ref=oauth2&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3D1a7cbb5eb1496643ff4631a25a73a3b3c6aca13d01813adab66f6a1f633a0c3d%26redirect_uri%3Dhttps%253A%252F%252Fen.gravatar.com%252Fconnect%252F%253Faction%253Drequest_access_token%26jetpack-code%26jetpack-user-id%3D0%26action%3Doauth2-login%26from-calypso%3D1&oauth2_client_id=1854). It should look as below:
<img width="1218" alt="截圖 2023-08-29 下午4 50 44" src="https://github.com/Automattic/wp-calypso/assets/21308003/77d241ae-de3a-46fb-aeb2-70e1ab670c21">
* Try to signup for a new account. The flow should work the same.

> Note: You can use [Gmail test accounts](https://www.eringibson.co/how-to-create-unlimited-testing-accounts-with-the-gmail-hack/) to test it.

## Pre-merge Checklist
